jwst/datamodels/schemas/miri_ifuptcor.schema.yaml

```diff
@@ -0,0 +1,82 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/fits-schema/fits-schema"
+id: "http://stsci.edu/schemas/jwst_datamodel/miri_ifuptcor.schema"
+title: MIRI IFU Point Source Spectral Corrections
+allOf:
+- $ref: referencefile.schema
+- type: object
+  properties:
+    data:
+      title: Photometric leak array
+      fits_hdu: SCI
+      default: 1.0
+      ndim: 2
+      datatype: float64
+    err:
+      title: Uncertainty array
+      fits_hdu: ERR
+      default: 0.0
+      ndim: 2
+      datatype: float64
+    dq:
+      title: Data quality array
+      fits_hdu: DQ
+      default: 0
+      ndim: 2
+      datatype: int64
```

Comment:
I believe most (if not all) DQ arrays up to this point are uint32. Is there a need for int64?

Reply:
No, not sure why that was set (values are all zero anyway).  I've changed to int32.